sdk_container/src/third_party/coreos-overlay/sys-kernel/coreos-modules/coreos-modules-6.12.54.ebuild

```diff
@@ -35,41 +35,32 @@ src_compile() {
 }
 
 src_install() {
+	local build="lib/modules/${KV_FULL}/build"
+
 	# Install modules to /usr.
-	# Install firmware to a temporary (bogus) location.
-	# The linux-firmware package will be used instead.
```

Comment:
Not a thing now?

Reply:
Oh yeah. The kernel dropped internal firmware and the `firmware_install` target (which might have been triggered by this?) in 2017.